doc/source/dev/conduct/code_of_conduct.rst

```diff
@@ -0,0 +1,162 @@
+SciPy Code of Conduct
+=====================
+
+
+Introduction
+------------
+
+This code of conduct applies to all spaces managed by the SciPy project,
```

Comment:
I counted, "SciPy community" is used twice and "community" without preceding "SciPy" seven times. It reads well, so prefer to keep it as is rather than start with a distracting sentence.